tidy3d/components/simulation.py

```diff
@@ -3235,15 +3235,14 @@ def make_adjoint_monitors(self, structure_indices: set[int]) -> tuple[list, list
         return adjoint_monitors_fld, adjoint_monitors_eps
 
     @property
-    def freqs_adjoint(self) -> list[float]:
+    def freqs_adjoint(self) -> set[float]:
```

Comment:
The return type here is still a list, `sorted(set)` converts the set to a sorted list.

Reply:
ah right, fixed. also used a `ValueError` instead of `assert`